src/main/java/org/dependencytrack/event/kafka/processor/VulnerabilityScanResultProcessor.java

```diff
@@ -904,6 +927,24 @@ ON CONFLICT ("VULNERABILITY_ID", "COMPONENT_ID", "PROJECT_ID") DO UPDATE
                 """)
         void createAnalysisComments(@BindMethods final Iterable<AnalysisComment> comment);
 
+        @SqlQuery("""
+                UPDATE "ANALYSIS"
```

Comment:
This still only updates an existing analysis, but doesn't create a new one if necessary.

Reply:
If I annotate it with `@SqlUpdate` it should return void, boolean, or Number but I need list of analysis to be returned.